SCF-Assignment-5/Question2/Searching.java

```diff
@@ -0,0 +1,67 @@
+package Question2;
+
+/**
+ * 
+ * this class includes function which will use to search element in array
+ * 
+ * @author Arjita
+ *
+ */
+public class Searching {
+	/**
+	 * this method is for linear search of element
+	 * 
+	 * @param array
+	 *            in which element is to be searched
+	 * @param x
+	 *            element which is to be searched
+	 * @param first

```

Comment:
Parameter name should be index as it is referring to array index which is getting checked